scripts/get-dns-zone-names.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+if [ $ENVIRONMENT = 'sbox' ]; then
+    ENVIRONMENT=sandbox
+fi
+
+if [ $PRODUCT = 'cft-platform' ]; then
+    if [ $ENVIRONMENT = 'stg' ]; then
+        ENVIRONMENT=aat
+    elif [ $ENVIRONMENT = 'test' ]; then
+        ENVIRONMENT=perftest
+    fi
+    APP_NAME=plum
+fi
+
+if [ $PRODUCT = 'sds-platform' ]; then
+    if [ $ENVIRONMENT = 'stg' ]; then
+        ENVIRONMENT=staging
+    fi
+    APP_NAME=toffee
+fi
+
+echo "##vso[task.setvariable variable=ENVIRONMENT;isOutput=true]true"
```

Comment:
I think it would be better not to override the environment name but instead refer to the converted one?

applies throughout
```suggestion
echo "##vso[task.setvariable variable=DNS_ENVIRONMENT;isOutput=true]true"
```